admin/installer/postgres/docker-compose-node1.yml

```diff
@@ -0,0 +1,12 @@
+postgres:
+  image: postgres
+  ports:
+     -  "5432:5432"
+  environment:
+     - "constraint:flocker-node==2"
+     - "POSTGRES_USER=flocker"
+     - "POSTGRES_PASSWORD=flocker"
```

Comment:
Let's create a followup issue to improve the security of the stack.
 * The nodes can all be created inside a VPC
 * With only private IP addresses
 * Flocker agents and control service can communicate internally using private IP addresses.
 * Moby docker app can access Postgres using private IP address
 * User can access Mobydocker web port using a floating / NAT public IP address.
 * User can access the client machine using a floating / NAT IP address
 * Might also make the stack cheaper by cutting down the need for precious IPv4 Addresses.